### PR TITLE
[Improvement][Task Plugin] Integrate zeppelin password authentication with DS zeppelin task plugin

### PR DESCRIPTION
**Pull request by xiaomin0322**
## Purpose of the pull request

add login zeppelin method

## Brief change log

org.apache.dolphinscheduler.plugin.task.zeppelin.ZeppelinTask
## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

**Review by EricGao888 — CHANGES_REQUESTED** [review]
Where do u plan to configure the zeppelin credentials? In web ui or in `common.properties`?
`parameters` is definitely not a good place to put those credentials, since it is a field to integrate with `Zeppelin Dynamic Form`.

**Comment by xiaomin0322** [comment]
@EricGao888  I expect to use the web UI to pass the parameters, and when I'm done, delete the parameters through zeppelinParamsMap.delete(). Is that OK with you?

**Comment by EricGao888** [comment]
> @EricGao888 I expect to use the web UI to pass the parameters, and when I'm done, delete the parameters through zeppelinParamsMap.delete(). Is that OK with you?

I think it is a good workaround to fill in the credentials in UI. You could refer to datasource center UI and try to mask the credentials in zeppelin task UI in the same way. Actually you don't need to call `zeppelinParamsMap.delete()` and it is fine if the credentials stay in memory as long as you don't print them in logging.

**Comment by xiaomin0322** [comment]
> > @EricGao888 I expect to use the web UI to pass the parameters, and when I'm done, delete the parameters through zeppelinParamsMap.delete(). Is that OK with you?
> 
> I think it is a good workaround to fill in the credentials in UI. You could refer to datasource center UI and try to mask the credentials in zeppelin task UI in the same way. Actually you don't need to call `zeppelinParamsMap.delete()` and it is fine if the credentials stay in memory as long as you don't print them in logging.

image

 already  use the web UI to pass the parameters, Please refer to the picture。

File: dolphinscheduler-ui/src/locales/zh_CN/project.ts
Comment (EricGao888) [inline dolphinscheduler-ui/src/locales/zh_CN/project.ts]:
```suggestion
    zeppelin_username: 'zeppelinUsername',
```

File: dolphinscheduler-ui/src/locales/zh_CN/project.ts
Comment (EricGao888) [inline dolphinscheduler-ui/src/locales/zh_CN/project.ts]:
```suggestion
    zeppelin_password: 'zeppelinPassword',
```

File: dolphinscheduler-task-plugin/dolphinscheduler-task-zeppelin/src/main/java/org/apache/dolphinscheduler/plugin/task/zeppelin/ZeppelinParameters.java
Comment (EricGao888) [inline dolphinscheduler-task-plugin/dolphinscheduler-task-zeppelin/src/main/java/org/apache/dolphinscheduler/plugin/task/zeppelin/ZeppelinParameters.java]:
```suggestion
    private String username;

```

File: dolphinscheduler-task-plugin/dolphinscheduler-task-zeppelin/src/main/java/org/apache/dolphinscheduler/plugin/task/zeppelin/ZeppelinParameters.java
Comment (EricGao888) [inline dolphinscheduler-task-plugin/dolphinscheduler-task-zeppelin/src/main/java/org/apache/dolphinscheduler/plugin/task/zeppelin/ZeppelinParameters.java]:
```suggestion
    private String password;

```

File: dolphinscheduler-ui/src/locales/en_US/project.ts
Comment (EricGao888) [inline dolphinscheduler-ui/src/locales/en_US/project.ts]:
```suggestion
    zeppelin_username: 'zeppelinUsername',
```

File: dolphinscheduler-ui/src/locales/en_US/project.ts
Comment (EricGao888) [inline dolphinscheduler-ui/src/locales/en_US/project.ts]:
```suggestion
    zeppelin_password: 'zeppelinPassword',
```

**Review by EricGao888 — COMMENTED** [review]
Please don't treat `password` as `pass word` and `username` as `user name`. BTW, please update related docs, both Chinese and English versions. Other LGTM.

**Comment by EricGao888** [comment]
> Please don't treat `password` as `pass word` and `username` as `user name`. BTW, please update related docs, both Chinese and English versions. Other LGTM.

There are still many such cases in this PR that I haven't marked, you need to fix them too : )

**Comment by xiaomin0322** [comment]
> > Please don't treat `password` as `pass word` and `username` as `user name`. BTW, please update related docs, both Chinese and English versions. Other LGTM.
> 
> There are still many such cases in this PR that I haven't marked, you need to fix them too : )

Modified relevant naming conventions and  related docs

**Review by EricGao888 — DISMISSED** [review]
LGTM, waiting for CI to pass

**Comment by EricGao888** [comment]
> LGTM, waiting for CI to pass

@xiaomin0322 You need to run "mvn spotless:apply" to fix formatting errors.

**Comment by xiaomin0322** [comment]
> > LGTM, waiting for CI to pass
> 
> @xiaomin0322 You need to run "mvn spotless:apply" to fix formatting errors.

Executed

**Review by EricGao888 — DISMISSED** [review]

**Comment by EricGao888** [comment]
> > > LGTM, waiting for CI to pass
> > 
> > 
> > @xiaomin0322 You need to run "mvn spotless:apply" to fix formatting errors.
> 
> Executed

@xiaomin0322 
image

Have u run it in ds root dir?

**Comment by xiaomin0322** [comment]
> > > > LGTM, waiting for CI to pass
> > > 
> > > 
> > > @xiaomin0322 You need to run "mvn spotless:apply" to fix formatting errors.
> > 
> > 
> > Executed
> 
> @xiaomin0322 image
> 
> Have u run it in ds root dir?
 is run it in ds root dir, Strange, local testing passed.

**Comment by EricGao888** [comment]
> > > > > LGTM, waiting for CI to pass
> > > > 
> > > > 
> > > > @xiaomin0322 You need to run "mvn spotless:apply" to fix formatting errors.
> > > 
> > > 
> > > Executed
> > 
> > 
> > @xiaomin0322 image
> > Have u run it in ds root dir?
> > is run it in ds root dir, Strange, local testing passed.
> > 

That's wired. It works on my laptop and I've fixed it for u.

**Review by EricGao888 — APPROVED** [review]
LGTM if CI passes.

**Comment by EricGao888** [comment]
@xiaomin0322 Merged. Thanks for your contributions : )

**Comment by xiaomin0322** [comment]
> > > > > > LGTM, waiting for CI to pass
> > > > > 
> > > > > 
> > > > > @xiaomin0322 You need to run "mvn spotless:apply" to fix formatting errors.
> > > > 
> > > > 
> > > > Executed
> > > 
> > > 
> > > @xiaomin0322 image
> > > Have u run it in ds root dir?
> > > is run it in ds root dir, Strange, local testing passed.
> > > 
> 
> That's wired. It works on my laptop and I've fixed it for u.

ths

**Pull request merged**
